src/api/spec/features/webui/projects/label_template_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'browser_helper'
+
+RSpec.describe 'LabelTemplates', :js, :vcr do
+  let!(:user) { create(:confirmed_user, :with_home, login: 'Jane') }
+  let(:project) { user.home_project }
+
+  before do
+    Flipper.enable(:labels)
+
+    login user
+  end
+
+  context 'when having no label templates' do
+    it 'creates a label template' do
+      visit project_label_templates_path(project)
+
+      click_on('Create Label Template')
+      fill_in('Name', with: 'A label template')
+      click_on('Create')
+
+      expect(page).to have_text('Label template created successfully')
```

Comment:
Why don't you test for the text of the `LabelTemplate` instead?

Reply:
When testing the delete action, it was not deterministic testing that the LabelTemplate was deleted, but using expectations on the page made capybara to wait until the page was fully loaded and made the test deterministic. For consistency reasons I rewrote the rest of the expectations to check for page content instead.

Reply:
I'm not quite sure what you mean. The page will contain the text `A label template` if you create a `LabelTemplate` with that name or not?

Reply:
Yes. And then I used the same method for the delete part, create a label template, hit the delete button and test that no label template was present anymore. But that test was flickering most of the time, so I switched from testing the presence or not of a label template to test the page content. Once I did that, I updated the create and edit expectations to check for page content, for consistency sake.

Reply:
We redirect during deletion and you expect on the page content, how can this flicker?

Reply:
I think it's an issue with the confirmation dialog for the delete, the expectation around the confirmation is very fast and I think capybara does not wait for the page to load again, so sometimes the label template is not yet deleted.

Reply:
Anyway, I'll change it.

Reply:
Done